src/lib/serviceRoutines/getContextEntitiesByEntityIdAndType.cpp

```diff
@@ -41,8 +43,18 @@
 *
 * getContextEntitiesByEntityIdAndType - 
 *
-* Invoked by:
-*  GET /v1/registry/contextEntities/type/{ETYPE}/id/{EID}
+* GET /v1/registry/contextEntities/type/{ETYPE}/id/{EID}
+*
+* URI parameters:
+*   - entity::type=XXX     (must coincide with entity::type in URL)
+*   - !exist=entity::type  (if set - error -- entity::type cannot be empty)
+*   - exist=entity::type   (not supported - ok if present, ok if not present ...)
```

Comment:
We don't support attributeFormat=object for the class ContextRegistrationAttribute.
That's why I didn't implement the URI option for the request.
However, we could support it ...
Not really a part of this PR anyway